### ISSUE TITLE
readme.md lost line with link in building site

### ISSUE THREAD

**Issue by dracodoc** [hadley/pkgdown#271]
I have [a package here](https://github.com/dracodoc/rCartoAPI). The first paragraph have a link 
<img width="916" alt="screen shot 2017-02-20 at 4 37 03 pm" src="https://cloud.githubusercontent.com/assets/1730496/23142602/d9de78be-f78a-11e6-8b47-4f9878c61d73.png">

Then the generated page is like this:

<img width="1046" alt="screen shot 2017-02-20 at 4 35 30 pm" src="https://cloud.githubusercontent.com/assets/1730496/23142567/a4133c24-f78a-11e6-9cb6-83952de87630.png">

Basically the first line is ignored, and the link was added to `Dev status`.

**Comment by hadley** [comment]
This by design. Most people use the first line for status badges
